indigo/distribution.yaml

```diff
@@ -1964,6 +1964,23 @@ repositories:
       url: https://github.com/ipa320/cob_supported_robots.git
       version: indigo_dev
     status: developed
+  cogrob_catkin_grpc:
```

Comment:
Since the package name is just `grpc` this name could also be `grpc`. It would make it easier for users calling `rosinstall_generator` since they need to use this name to identify which repos to fetch.

If you update this line the following lines in the `release` section will need to be removed since the single package name matches the repo name now:

```
packages:
- grpc
```

Reply:
Thanks for the comments. PTAL.